origin.spec

```diff
@@ -152,6 +153,9 @@ Provides:       tuned-profiles-%{name}-node
 Summary:        %{product_name} Client binaries for Linux
 Obsoletes:      openshift-clients < %{package_refactor_version}
 Requires:       bash-completion
+# Provided by containers-common or skopeo-containers, depending on the release.
+Requires:       /etc/containers/registries.conf
```

Comment:
The client binaries don't require these, so no, we wouldn't add them.

Reply:
`oc` will not be allowed to `Require` `docker` or `crio` or `podman`.

Reply:
so do we need to move our builder logic out of oc then? because right now that's how it works, right?

ie "origin-docker-builder" is just based on "origin-cli" and runs the oc binary via symlink as "openshift-docker-build"

Reply:
We discussed making this available through a config map to buildah. Does that work?

Reply:
if we have a concept of a global configmap that any pod can mount, yes.

Reply:
though you'll also have to solve who's creating that configmap

Reply:
@mrunalp yeah, I think it'll work.  We can provide default files in the image, but allow any/all of `storage.conf`, `registries.conf`, and `policy.json` as files in a config map (`oc create configmap build-system-configs --from-file=/etc/containers`, or the like) to override the version in the image.